modules/manage/pages/audit-logging.adoc

```diff
@@ -23,14 +23,15 @@ Messages recorded to the audit log topic comply with the https://schema.ocsf.io/
 
 Redpanda's audit logging mechanism supports several options to control the volume and availability of audit records. Configuration is applied at the cluster level using the standard https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-property-configuration/[cluster configuration mechanism]. The cluster configuration options exclusive to audit logging include the following:
 
-* `audit_enabled`: Boolean value to enable audit logging. When you set this to true, Redpanda checks for an existing topic named `__audit_log`. If none is found, Redpanda automatically creates one for you. Default: `false`.
-* `audit_log_num_partitions`: Integer value defining the number of partitions used by a newly created audit topic. This configuration applies only to the audit log topic and may be different from the cluster or other topic configurations. This cannot be altered for existing audit log topics. Default: `12`.
-* `audit_log_replication_factor`: Integer value defining the replication factor for a newly created audit log topic. This configuration applies only to the audit log topic and may be different from the cluster or other topic configurations. This cannot be altered for existing audit log topics. Default: `3`.
-* `audit_client_max_buffer_size`: Integer value defining the number of bytes allocated by the internal audit client for audit messages. When changing this, you must disable audit logging and then re-enable it for the change to take effect. Consider increasing this if your system generates a very large number of audit records in a short amount of time. Default: `16MiB`.
-* `audit_queue_max_buffer_size_per_shard`: Integer value defining the maximum amount of memory used by the audit buffer in each shard. Once this size is reached, requests to log additional audit messages will return a non-retryable error. You must restart the cluster when changing this value. Default: `1MiB`.
-* `audit_enabled_event_types`: List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following - `management, produce, consume, describe, heartbeat, authenticate, schema_registry, admin`. Default: `'["management","authenticate","admin"]'`.
-* `audit_exclude_topics`: List of strings in JSON style identifying the topics the audit logging system should ignore. This list cannot include the `__audit_log` topic. Redpanda will reject the command if you do attempt to include that topic. Default: null.
-* `audit_exclude_principals`: List of strings in JSON style identifying the principals the audit logging system should ignore. Principals can be listed as `User:name` or `name`, both are accepted. Default: null.
+* xref:reference:cluster-properties.adoc#audit_enabled[`audit_enabled`]: Boolean value to enable audit logging. When you set this to true, Redpanda checks for an existing topic named `__audit_log`. If none is found, Redpanda automatically creates one for you. Default: `false`.
```

Comment:
FYI, there was a last minute request from Alex to adjust the naming of certain internal topics, which included the audit log (PR: https://github.com/redpanda-data/redpanda/pull/15817).

So the name of the audit log is no longer `__audit_log` it is `_redpanda.audit_log`